protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3890,7 +3890,7 @@ message OrderInfo {
   bool is_market_order = 4;
 }
 
-// Request for the estimate of the margin level and, if available collateral is provided, liquidation price for the specified position.
+// Request for the estimate of the margin level, margin account balance change estimate and liquidation price for the specified position.
```

Comment:
```suggestion
// Request for the estimated margin level, margin account balance change, and liquidation price for the specified position.
```

All 3 are estimates, right? It's clearer if either all 3 say "estimated" or there's just one use of it to cover all 3 values.

Reply:
Yes, thanks, I've applied all the suggestions.